src/main/resources/application.properties

```diff
@@ -12,4 +12,18 @@ red.jambot.soundboard.sound-base-url=https://jospiek.nl/soundboard/audio/mp3/
 
 spring.application.name=jambot
 
+management.metrics.tags.env=production
+
 management.endpoints.web.exposure.include=health,metrics
+
+spring.datasource.url=jdbc:mariadb://localhost:3306/jambot
```

Comment:
Is even kijken hoe we dit gaan deployen, wss gewoon docker file bijwerken :).

Hoe heb je lokaal gedaan?

Reply:
For local development, I'm using the method described in the getting started documentation. So with the application-local.properties and my own credentials of the DB.